### DISCUSSION TITLE
MinerX.3 1.13.2 General Testing and Call Outs

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#7719] [Show and tell]
Same old same old. Update your system to lotus [v1.13.2-rc1](https://github.com/filecoin-project/lotus/releases/tag/v1.13.2-rc1).

❤️ This post if you are participating in this testing round. If you aren't able to join for reasons, please reach out to @ Deep Kapur and me on slack!

## Regular Operations 

Your daily SP operations should stay ✅ .

**Reply with:**
🦭 If you have successfully sealed some new sectors.
🤝 If you have successfully made some storage deals.
💻 If you passed your first WindowPoSt.

❗ Create [an issue](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fbug&template=bug_report.yml) if you think there is a bug, don't forget to attach the logs!
   - Save the daemon logs, miner logs, and `lotus-miner info all` if you want to report a bug! If the log files are large, you can upload them in `MinerX.3 - v1.13.2 Testing Logs` folder and link it in your issue report/slack message/..

## New feature and Improvements 

Except for worker and scheduler enhancements and retrieval improvements, you should also watch out for:
- You should notice that cross worker tar transfers use less CPU, and might be faster on 10G+ networks, please leave a comment here if that's something you observed! 
-  Try `lotus-shed balancer` out, it should help you with multi wallet management, for example between worker and control addresses!
-  wdpost should start ~10 epoch earlier.
- Play around with `lotus-shed msg`, it's getting really handy for inspecting msig messages (which contains internal invocations).
- Inspect your faults by running `lotus-shed miner faults` to get a list of faulty sectors and the expiring times.

**Comment by stuberman** [comment]
I am rejecting all deals until the pipeline is clear, which should be Wed 1 Dec am. then will upgrade and accept deals.

**Reply by stuberman** [reply]
`lotus-miner version`
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242

**Reply by stuberman** [reply]
💻 PoSTs work 
Market node crashed upon offline import - will post the issue

**Reply by stuberman** [reply]
Just mined a block...

> 2021-12-01T16:31:06.458Z	INFO	storageminer	storage/miner.go:290	Computing WinningPoSt ;[{SealProof:8 SectorNumber:2103 SealedCID:bagboea4b5abcbadt7u33svnaydcc3ft7g57a26r5bgqbybqd3cw6epstfnc7js3e}]; [105 246 44 25 241 231 88 220 102 186 140 104 34 215 42 131 102 208 119 146 126 56 102 39 215 67 19 79 39 200 49 199]

**Reply by stuberman** [reply]
🤝 Online and offline deals working well
🦭 Sealed >10 sectors

**Comment by kernelogic** [comment]
I yolo-ed and now I'm on the new version.

lotus-miner version
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242

So far windowpost OK
CC OK
Online deals OK

Update: winningpost OK!!

**Comment by TippyFlitsUK** [comment]
```
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242
```
🤝🦭💻

**Comment by TippyFlitsUK** [comment]
I'm using `Netdata` on my server and after installing this release I started to receive repeated `1m_ipv4_udp_send_buffer_errors` warning messages. (Ubuntu 20.04.3 LTS)

I'm guessing that it probably has something to with the `Use 1M buffers for Tar transfers` in this new release.

I have updated my buffer limits using the steps [on this page](https://medium.com/@CameronSparr/increase-os-udp-buffers-to-improve-performance-51d167bb1360) and the warning messages have now stopped.

Might be a good idea for miners to double check their buffer limits. I didn't see any obvious errors with transfers before upping my limits but I'm not sure if it would have had an impact over a longer period of time.

NOTE: You may need to restart your miner after updating the limits as doing it on the fly caused Lotus sync issues for me.

**Comment by NSC-FIL** [comment]
Upgraded f01045784 to:
lotus version 1.13.2-rc1+mainnet+git.52c7fa242
lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242
Once its chain is pruned, it will be ready to go by this afternoon.

**Reply by NSC-FIL** [reply]
Successfully sealed some new sectors.
Successfully made some storage deals.
Passed first WindowPoSt.

**Comment by Angelo-gh3990** [comment]
`##lotus version (says api 1.4)`
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.4.0
Local: lotus version 1.13.2-rc1+mainnet+git.52c7fa242

`##lotus-miner version (says api 1.2)`
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242

workers are upgraded too, seems to be running fine ... Will have the WinPoST check soon and sealing is starting up as we speak.

**Reply by Angelo-gh3990** [reply]
WinPoST, sealing, etc works fine. No block win yet

**Reply by Angelo-gh3990** [reply]
No retrieval test yet

**Comment by Meest-NAM** [comment]
Meest NAM (f03488)

lotus-miner version
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242

lotus version
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.4.0
Local: lotus version 1.13.2-rc1+mainnet+git.52c7fa242


So far wdpost OK
CC OK
Online deals OK
winningPost OK!!

**Comment by Reiers** [comment]
**The whole production line is upgraded.** 

```
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.4.0
Markets: 1.13.2-rc1+mainnet+git.52c7fa242+api1.4.0
Miner: 1.13.2-rc1+mainnet+git.52c7fa242+api1.4.0
Local: lotus version 1.13.2-rc1+mainnet+git.52c7fa242
```

**Reply by Reiers** [reply]
- [x] windowPost - Status: ok
- [x] winningPost - Status: ok
- [x] test CC sector (AP,PC1,PC2,C2) - Status: ok
- [x] test online deal - Status: ok 
- [x] test offline deal - Status: ok
- [ ] test retrieval deal - Status: Pending

**Reply by Reiers** [reply]
🦭
🤝
💻

**Comment by rjan90** [comment]
After initial testing: Everything seems fine. 🤝🦭💻 without problems.
```
lotus version
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.4.0
Local: lotus version 1.13.2-rc1+mainnet+git.52c7fa242
lotus-miner version
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242
```

**Reply by rjan90** [reply]
- Inspected some random storage providers faults with `lotus-shed miner faults`. And initial feel is that it works properly on those who run the stock lotus code. Getting some weird outputs on large storage providers who most likely run custom code. 
- Have not tested `lotus-shed balancer` in production yet. But after some initial issues with how to call the command, it seems like `lotus-shed balancer "provide;addr1;minimumFIL" "request;addr2;low;high"` is the correct way of calling it.
- `./lotus-shed msg` works really well, and gives a LOT more detailed information on _Propose-messages_ compared to online chain-explorers currently.

**Comment by neondragon** [comment]
🦭
🤝
💻

**Comment by benjaminh83** [comment]
Passed windowPoST and a couple of sealed sectors. Everything is normal... 
```
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242
```
I said I would wait for next week on the scheduler optimizations, but then @magik6k just told we can do: 
C2_32G_GPU_UTILIZATION=0.5 
PC2_32G_GPU_UTILIZATION=0.5
Hopefully allowing a worker to share a GPU between jobs. That would be AWESOME!!!!!! I have to test!

**Comment by benjaminh83** [comment]
So from here: 
🦭 
🤝
💻

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
all on 1.13.2-rc1, taking deals, wd post in 12:30h - update when results are in

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
deals work
sealing works

no idea how smilies work here

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
wd post works

**Comment by ksidles** [comment]
All upgraded to: lotus version 1.13.2-rc1+mainnet+git.52c7fa242
🦭
🤝
💻

**Comment by kenshyx** [comment]
Finished also the upgrade to v1.13.2-rc1
```
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242
```

**Reply by kenshyx** [reply]
🤝  💻 🦭

**Comment by tmyuu** [comment]
### Version

```
Daemon:  1.13.2-rc1+mainnet+git.52c7fa242+api1.4.0
Local: lotus version 1.13.2-rc1+mainnet+git.52c7fa242
```

### Status
* [x]  windowPost - Status: ok
* [x]  winningPost - Status: ok
* [x]  test CC sector (AP,PC1,PC2,C2) - Status: ok
* [x]  test online deal - Status: ok
* [x]  test offline deal - Status: ok
* [x]  test retrieval deal - Status: ok

**Comment by William8Work** [comment]
I want to report an issue here. I have two AP+PC1 workers. After upgrade to 1.13.2-rc1, one of the workers are normal, but the other worker performance got slow down drastically.

Ubuntu 20.04 with latest updates
one Epyc 7F32 CPU with 16 cores
256GB RAM, with another 256GB swap on Nvme

The PC1 worker having problem is with 7F32 CPU. Prior to the upgrade, when running 1.13.0 or prior versions, it runs 6 parallel PC1 jobs and can finishes around 4 hours. With 1.13.2-rc1, it took over 20 hours to do that. Turns out if I added FIL_PROOFS_USE_MULTICORE_SDR=1 to this PC1 worker, the performance per PC1 is not only back but slightly better than 1.13.0 level, however, looks like it only seal max 4 sectors in parallel probably due to the ccx limits.

I didn't set FIL_PROOFS_SDR_PARENTS_CACHE_SIZE, not sure if it makes a difference.

Does anyone else see PC1 performance differences with SDR=0?

**Comment by marco-storswift** [comment]
version
Daemon: 1.13.2-rc1+mainnet+git.52c7fa242+api1.2.0
Local: lotus-miner version 1.13.2-rc1+mainnet+git.52c7fa242

 windowpost OK
CC OK
Online deals OK

Update: winningpost OK!!

**Reply by Reiers** [reply]
**Thanks for the report ! but this is MinerX testing.**

If you want to test any release candidates on your own, always upgrade to the latest tags:
https://github.com/filecoin-project/lotus/releases/tag/v1.13.2-rc7

We are now on rc7 as you can see :)

**Reply by jennijuju** [reply]
Haha he is one of the fellows - but certainly on an older rc lolol

**Reply by Reiers** [reply]
my bad - sorry about that.
Either way - you should upgrade to the latest rc :)
